1.8/administration/virtual-networks/ip-per-container.md

```diff
@@ -0,0 +1,194 @@
+```
+
+**Note:** To use virtual networks in DC/OS you MUST make sure to use a recent Linux kernel (3.9 or above) as well as Docker version 1.11 on the agent nodes.
+
+Each overlay network is identified by a canonical `name`. Containers launched on an overlay network will get an IP address from the `subnet` allocated to the overlay network. To remove the dependency on a global IPAM (IP address management system), the overlay `subnet` is further split into smaller subnets, with each subnet being allocated to an agent. The agents can then use a host-local IPAM to allocate IP addresses from their respective subnets to containers launched on the agent and attached to the given overlay. The `prefix` determines the size of the subnet (carved the overlay `subnet`) allocated to each
+agent.
+
+In the configuration above, each virtual network is allocated a /17 subnetwork (in the “subnet” field), which is then divided in /24 subnetworks to be used in each host that will be part of the network (in the “prefix” field). This allocates 8 bits (32 total minus 24 allocated for “prefix”) to designate the endpoint (container) inside a host for a maximum of 255 endpoints per host. It also allocates 7 bits (24 minus 17) for hosts that will be members of this overlay network, which provides a maximum of 127 hosts. These values can be configured to adapt to each installation’s needs.
+
+# Adding overlay networks during installation
+
+Currently (as of version 1.8), DC/OS virtual networks can only be added and configured at install time. You can override the default network or add additional virtual networks by modifying your `config.yaml` file:
+
+```yaml
+    agent_list:
+    - 10.10.0.117
+    - 10.10.0.116
+    # Use this bootstrap_url value unless you have moved the DC/OS installer assets.
+    bootstrap_url: file:///opt/dcos_install_tmp
+    cluster_name: &lt;cluster-name&gt;
+    master_discovery: static
+    master_list:
+    - 10.10.0.120
+    - 10.10.0.119
+    - 10.10.0.118
+    resolvers:
+    - 8.8.4.4
+    - 8.8.8.8
+    ssh_port: 22
+    ssh_user: centos
+    dcos_overlay_network:
+      vtep_subnet: 192.15.0.0/20
+      vtep_mac_oui: 70:B3:D5:00:00:00
+      overlays: 
+        - name: dcos-1
+          subnet: 192.168.0.0/17
+          prefix: 24
+        - name: dcos-2
+          subnet: 192.168.128.0/17
+          prefix: 24
+        - name: dcos-3
+          subnet: 44.128.0.0/16
+          prefix: 24
+```
+
+In the above example, we have defined three overlay networks. The overlay networks `dcos-1` and `dcos-2` basically retain the default overlay networks, and we have added another overlay network called `dcos-3` with subnet range `44.128.0.0/16`. When you create a network, you must give it a name and a subnet. That name is used to launch Marathon tasks and other Mesos frameworks tasks using this specific overlay network.
+
+**Note:** If you specify the same prefix and network mask for an overlay network, it will only work on one agent node.
```

Comment:
The prefix defines the number of Agents on which the overlay can run. This is already explained in the previous section (I think), so seems like this NOTE is actually redundant?